src/diff.rs

```diff
@@ -1337,9 +1373,44 @@ impl DiffFindOptions {
     // TODO: expose git_diff_similarity_metric
 }
 
+impl Default for DiffFormatEmailOptions {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl DiffFormatEmailOptions {
+    /// Creates a new set of email options,
+    /// initialized to the default values
+    pub fn new() -> Self {
+        let mut opts = DiffFormatEmailOptions {
+            raw: unsafe { mem::zeroed() },
+        };
+        assert_eq!(
+            unsafe { raw::git_diff_format_email_options_init(&mut opts.raw, 1) },
+            0
+        );
+        opts
+    }
+
+    fn flag(&mut self, opt: u32, val: bool) -> &mut Self {
+        if val {
+            self.raw.flags |= opt;
+        } else {
+            self.raw.flags &= !opt;
+        }
+        self
+    }
+
+    /// Exclude `[PATCH]` from the subject header
+    pub fn exclude_subject_patch_header(&mut self, should_exclude: bool) -> &mut Self {
+        self.flag(raw::GIT_DIFF_FORMAT_EMAIL_EXCLUDE_SUBJECT_PATCH_MARKER, should_exclude)
+    }
```

Comment:
Would you be interested in expanding this to include other options too?

Reply:
As of right now, these are [the only flags implemented](https://github.com/libgit2/libgit2/blob/fb439c9/include/git2/diff.h#L1367-L1374)